tachyon/math/finite_fields/quadratic_extension_field_unittest.cc

```diff
@@ -145,7 +145,7 @@ TEST_F(QuadraticExtensionFieldTest, MultiplicativeOperators) {
     GF7_2 tmp = test.a;
     tmp *= test.b;
     EXPECT_EQ(tmp, test.mul);
-    tmp /= test.b;
+    ASSERT_TRUE(tmp /= test.b);
```

Comment:
I'm not sure if we can always assure the quotient to have value.

Reply:
`test.b` is guaranteed to be `{GF7(3), GF7(5)}`